src/ngraph/builder/split.cpp

```diff
@@ -74,3 +99,24 @@ NodeVector builder::split(const Output<Node>& value, size_t split_parts, int axi
     std::vector<size_t> length_parts(split_parts, length_axis_to_split / split_parts);
     return split(value, length_parts, axis_to_split);
 }
+
+NodeVector builder::opset1::split(const Output<Node>& value,
+                                  const std::vector<size_t>& split_lengths,
+                                  int64_t axis)
+{
+    const auto axis_node = op::Constant::create(element::u64, Shape{}, {axis});
```

Comment:
Let's stick to one convention here.
```suggestion
    const auto axis_node = ngraph::opset1::Constant::create(element::u64, Shape{}, {axis});
```

Reply:
done